apps/sms/locales/sms.en-US.properties

```diff
@@ -135,8 +135,18 @@ discard-new-message         = Discard the message you’re composing and open th
 resend-confirmation         = The message could not be sent. Try again?
 resend-message              = Resend this message
 deleteMessage-confirmation  = Delete selected message?
-deleteMessages-confirmation = Delete selected messages?
-deleteThreads-confirmation2 = Delete selected message threads?
+deleteMessages-confirmation-message = {[ plural(n) ]}
+deleteMessages-confirmation-message[one] = Delete selected message?
```

Comment:
You're missing a [zero] case. Even if not actually used in the code, you need the string to make external localization tools happy (they expect to find all plural forms).